EmptyImportSubUsersFile.php

```diff
@@ -32,9 +32,6 @@
 
 if (Tools::getValue('proc_success') != '') {
     $handle = fopen(_PS_MODULE_DIR_ . 'sendinblue/csv/ImportSubUsersToSendinblue.csv', 'w');
-    $key_value = array();
-    $key_value[] = '';
-    fputcsv($handle, $key_value, '');
```

Comment:
This code is used to write blank data after processing as we don't want any once to access these data by using some URL.